### ISSUE TITLE
[DS-META]  Type vs "Kind" of Data

### ISSUE THREAD

**Issue by SDooman** [bootstrapworld/curr#235]
Something that's been at the back of my mind for a while is the potential confusion between classifying data as Quant vs Categorical, and identifying the Type of data in a column.  In plain english, it's reasonable to answer question "what type of data is in this column" by saying "quantitative" or "Number".  

It need a nice word that **isn't** Type to refer to the classification of data in a column.  Maybe classification?  Not great since it's very long.

**Comment by schanzer** [comment]
"Kind" is good enough for now. Make it so.

**Comment by schanzer** [comment]
Unit 2 edited to use Kind.
